lib/src/androidTest/java/tech/relaycorp/relaydroid/persistence/EncryptedDiskPersistenceTest.kt

```diff
@@ -0,0 +1,79 @@
+package tech.relaycorp.relaydroid.persistence
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.test.TestCoroutineScope
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.nio.charset.Charset
+
+@RunWith(AndroidJUnit4::class)
+class EncryptedDiskPersistenceTest {
+
+    private val coroutineScope = TestCoroutineScope()
+    private val subject =
+        EncryptedDiskPersistence(
+            ApplicationProvider.getApplicationContext(),
+            coroutineScope.coroutineContext
+        )
+
+    @After
+    fun tearDown() {
+        coroutineScope.runBlockingTest {
+            subject.deleteAll()
+        }
+    }
+
+    @Test
+    fun getAndSet() = coroutineScope.runBlockingTest {
+        assertNull(subject.get("file"))
+        val data = "test"
+        subject.set("file", data.toByteArray())
+        assertEquals(data, subject.get("file")?.toString(Charset.defaultCharset()))
+    }
+
+    @Test
+    fun delete() = coroutineScope.runBlockingTest {
```

Comment:
Can you please nest tests for get and add a test for when the file doesn't exist?